qiskit/tools/visualization.py

```diff
@@ -571,87 +569,84 @@ def plot_wigner_function(state, res=100):
         xaxis (np.array):  the range of the x axis
-        
+
     Returns:
         none: plot is shown with matplotlib to the screen
-        
     """
-    if xaxis is None:
-        xaxis = np.linspace(0,len(wigner_data)-1,num=len(wigner_data))
-    
-    plt.plot(xaxis,wigner_data)
+    if not xaxis:
+        xaxis = np.linspace(0, len(wigner_data)-1, num=len(wigner_data))
+
+    plt.plot(xaxis, wigner_data)
     plt.show()
 
-def plot_wigner_plaquette(wigner_data, maxWigner='local'):
+
+def plot_wigner_plaquette(wigner_data, max_wigner='local'):
     """Plots plaquette of wigner function data, the plaquette will
     consist of cicles each colored to match the value of the Wigner
     function at the given point in phase space.
-        
+
     Args:
         wigner_data (matrix): array of Wigner function data where the
                             rows are plotted along the x axis and the
                             columns are plotted along the y axis
-        maxWigner:  - 'local' puts the maximum value to maximum of the points
+        max_wigner:  - 'local' puts the maximum value to maximum of the points
                     - 'unit' sets maximum to 1
                     - float for a custom maximum.
-        
+
     Returns:
         none: plot is shown with matplotlib to the screen
-        
     """
     wigner_data = np.matrix(wigner_data)
     dim = wigner_data.shape
-    
-    if maxWigner == 'local':
-        Wmax = np.amax(wigner_data)
-    elif maxWigner == 'unit':
-        Wmax = 1
+
+    if max_wigner == 'local':
+        w_max = np.amax(wigner_data)
+    elif max_wigner == 'unit':
+        w_max = 1
     else:
-        Wmax = maxWigner #For a float input
-    
-    cmap = matplotlib.cm.get_cmap('seismic_r')
-    
+        w_max = max_wigner  # For a float input
+
+    cmap = plt.cm.get_cmap('seismic_r')
+
     xax = dim[1]-0.5
     yax = dim[0]-0.5
     norm = np.amax(dim)
-    
-    fig = plt.figure(figsize=((xax+0.5)*6/norm,(yax+0.5)*6/norm))
+
+    fig = plt.figure(figsize=((xax+0.5)*6/norm, (yax+0.5)*6/norm))
     ax = fig.gca()
-    
+
     for x in range(int(dim[1])):
         for y in range(int(dim[0])):
-            
-            circle = plt.Circle((x,y),0.49,
-                                color =
-                                cmap((wigner_data[y,x]+Wmax)/(2*Wmax)))
+            circle = plt.Circle(
+                (x, y), 0.49, color=cmap((wigner_data[y, x]+w_max)/(2*w_max)))
             ax.add_artist(circle)
 
     ax.set_xlim(-1, xax+0.5)
     ax.set_ylim(-1, yax+0.5)
     ax.set_xticks([], [])
     ax.set_yticks([], [])
     m = cm.ScalarMappable(cmap=cm.seismic_r)
-    m.set_array([-Wmax, Wmax])
+    m.set_array([-w_max, w_max])
     plt.colorbar(m, shrink=0.5, aspect=10)
     plt.show()
 
-def plot_wigner_data(wigner_data, phis=None,
-                     thetas=None, method=None,
+
+def plot_wigner_data(wigner_data, phis=None, thetas=None, method=None,
```

Comment:
In a similar note to a comment above, can you check if the parameter `theta` is needed, as it seems to be unused?